site/en/blog/prerender-pages/index.md

```diff
@@ -0,0 +1,263 @@
+
+{% Aside %}
+  Prerendered pages should be treated as a hint and a progressive enhancement. Unlike `preload`, prerender is a speculative _hint_ and the browser may choose not to act upon the hint based on user settings, current memory usage, or other heuristics.
+{% endAside %}
+
+### Impact of prerendering
+
+Prerendering allows a near-instant page load as shown in the Video below when prerender was enabled on [web.dev](https://web.dev/), where the first blog article is prerendered:
+
+{% Video src="video/W3z1f5ZkBJSgL1V1IfloTIctbIF3/rO1MGC5jaBAIyo79KrVT.mp4",
+  controls="true",
+  loop="true",
+  muted="true",
+  playsinline="true",
+  poster="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/zFNgVLiXhoPWpuveGPgY.png"
+%}
+
+Web.dev is already a fast site, but even with this we can see how prerendering improves the user experience. This can therefore also have a direct impact on a site’s [Core Web Vitals](https://web.dev/vitals/), with near zero LCP, reduced CLS (since any load CLS happens before the initial view), and improved FID (since the load should be completed before the user interacts).
+
+Even when a page is not fully prerendered, having a head start to the page load, should improve the loading experience. When a link is activated while prerendering is still happening, the page prerendering page will move to the main frame and continue loading.
+
+However, prerendering does use additional memory and network bandwidth. Care must be taken not to over-prender, at a cost of user resources. Only when there is a high likelihood of the prerender page being navigated to, should the page be prerendered.
+
+See the [Measuring performance](#measuring-performance) section for more information on how to measure the actual performance impact in your analytics.
+
+### Viewing Chrome’s omnibox predictions
+
+For the first two use cases, you can view Chrome’s predictions for URLs in the `chrome://predictors` page:
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/Oy3gv9Gyhx0eTKWVtUVo.png", alt="Screenshot of the Chrome Predictors page filtered to show low (grey), medium (amber), and high (green) predictions based on text entered.", width="800", height="560" %}
+
+Green lines indicate confidence high enough to trigger prerendering. In this example typing “s” gives a reasonable confidence (amber), but once you type “sh” then Chrome has enough confidence that you nearly always navigate to [https://sheets.google.com](https://sheets.google.com).
+
+This screenshot was taken in a relatively fresh Chrome install, and filtering out zero confidence predictions, but if you view your own predictors you will likely see considerably more entries, and potentially more characters required to reach a high enough confidence level.
+
+These predictors are also what drive the Omnibox suggested options you may have noticed:
```

Comment:
Is omnibox a proper noun? If so, it should be consistently capitalized (or uncapitalized) throughout the post.

Reply:
I'm resolving these to lowercase throughout.